linera-service/src/cli_wrappers/kind.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::util::CommandExt;
+use anyhow::Result;
+use rand::Rng;
+use tokio::process::Command;
+
+pub struct KindCluster {
+    id: u32,
+}
+
+impl KindCluster {
+    pub fn new(cluster_id: Option<u32>) -> Self {
```

Comment:
Same comment. With this choice of API, we can have a `KindCluster` value that is not created yet.

Reply:
So this one is harder because everytime we create a `kind` cluster, `kubectl` will change context to that cluster. So we will have to manually switch contexts here. More work for me, but I guess the code looks clearer on what's actually happening 🤔 gonna do that in a sec

Reply:
Ah actually this won't be an issue until the next PR I'm working on, so I'll with that there :)